shared/models/git_repo.rb

```diff
@@ -161,13 +177,13 @@ def git
     # call like you would, but you also get the git repo involved, so it's  .each { |git, branch| branch.yolo; git.yolo }
     def git_and_remote_branches_each(&each_block)
       git_action_with_queue do
-        super_verbose("iterating through all remote branches of #{self.git_config.git_url}")
+        logger.debug("iterating through all remote branches of #{self.git_config.git_url}")
         branch_count = 0
         self.git.branches.remote.each do |branch|
-          each_block.call(self.git, branch)
-          branch_count = branch_count + 1
+          yield(self.git, branch)
```

Comment:
I actually prefer `each_block` here to be a little more explicit about what is going on. I think `yield` may be more preferred, but in cases like this I prefer clarity.